### ISSUE TITLE
Did you retrain the method in paper"...Automatic Portrait Segmentation...", when you get the background mask?

### ISSUE THREAD

**Issue by SwordHolderSH** [yiranran/APDrawingGAN#6]
Dear authors:
    This is an outstanding work, but here is a small problem.
    1) In "Preprocessing steps>Prepare background masks" of APDrawingGAN, we run the code of paper "Automatic Portrait Segmentation for Image Stylization" obtained from the link "http://xiaoyongshen.me/webpage_portrait/index.html". 
    2) We use the pre-trained models proved by authors to test the code (we did not change anything in the codes), but it cannot achieve the effect of the paper,  our test results as shown in the following Figures:
![00321](https://user-images.githubusercontent.com/31804639/62843775-1d856d80-bcef-11e9-9f5f-11c27c8a0be5.jpg)
![2_aligned](https://user-images.githubusercontent.com/31804639/62843776-1e1e0400-bcef-11e9-8c3e-ea0a3b18fca9.png)
![1_aligned](https://user-images.githubusercontent.com/31804639/62843777-1e1e0400-bcef-11e9-9a6a-f04e9d073c63.png)
    3) Could you please tell me how you employ their code, whether you retrain it, and so on?
    
    Very thanks!

**Comment by SwordHolderSH** [comment]
We have obtained background-masks by Baidu-Api instead of "Automatic Portrait Segmentation for Image Stylization" , thanks you!

**Comment by ichuniq** [comment]
Hi @DoctorSH, 
For your information or others who may similar doubts, I tested the model using the pre-trained model and the Matlab code `testing/demo_portraitFCN.m` which are both provided in [Automatic Portrait Segmentation for Image Stylization](http://xiaoyongshen.me/webpage_portrait/index.html), the results seems to be ok under most circumstances. It didn't do well when there is a small color difference between people and the background or when the image is overexposed.
I believe there are other advanced segmentation models available nowadays, so if the pre-trained one doesn't meet your expectation when preprocessing your dataset, try other models or retrain it by yourself, but in the scope of frontal, ID-like photo, which APDrawingGAN is trained in, I think it is good enough.


![rsz_masked0321](https://user-images.githubusercontent.com/47471390/63038681-4aef3880-bef4-11e9-9c7b-a56d2f893777.jpg)
